build/release/containers.sh

```diff
@@ -75,22 +75,21 @@ ENTRYPOINT ["/usr/bin/rookd"]
 EOF
     build_artifact $os $arch $tmpdir rook/rookd rookd
     rm -fr $tmpdir
-
-    # TODO: build the client and toolbox for arm
-    [[ ${arch} == "amd64" ]] || return 0
  
     echo "Building the rook-client container"
     mkdir $tmpdir
     cat <<EOF > $tmpdir/Dockerfile
-FROM ubuntu
-RUN apt-get update && \
-    apt-get install -yq --no-install-recommends wget s3cmd kmod module-init-tools sudo
+FROM ${baseimage}:3.5
+RUN apk add --no-cache e2fsprogs sudo
```

Comment:
i believe the only place now is for modprobe
https://github.com/travisn/rook/blob/alpine-client/pkg/util/sys/kmod.go#L31